### PR TITLE
[MIG] project_task_code: Migrate to 10.0

### PR DESCRIPTION
**Pull request by kellylougheed**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Why don't you start from the 9.0 version using migration guide: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0?

**Comment by kellylougheed** [comment]
Will do @pedrobaeza -- thanks!

**Comment by pedrobaeza** [comment]
No need to close this. You can just push force with the same branch name if you want

**Comment by lasley** [comment]
Hah yeah it was an auto-close after a reset then a failed format-patch & a subsequent push -f, we're working on the git wizardry right now 👍

**Comment by lasley** [comment]
Awwwww but it didn't reopen when the branch was pushed to again. Shame Github, Shame! 

@kellylougheed can you reopen from your end?

**Comment by dreispt** [comment]
AFAICT you're basically proposing for v10 the changeset merged on v9, correct?

**Comment by kellylougheed** [comment]
@dreispt Yes, this should just be an upgrade to v10

File: project_task_code/__manifest__.py
Comment (dreispt) [inline project_task_code/__manifest__.py]: 10.0.1.0.0

File: project_task_code/README.rst
Comment (dreispt) [inline project_task_code/README.rst]: s/9/10

File: project_task_code/README.rst
Comment (dreispt) [inline project_task_code/README.rst]: s/project-service/project

**Review by dreispt — CHANGES_REQUESTED** [review]
LGTM expect for some small nitpicks

File: project_task_code/views/project_view.xml
Comment (dreispt) [inline project_task_code/views/project_view.xml]: Are you sure about this one? AFAICT `string` was correct.

Comment (kellylougheed) [reply project_task_code/views/project_view.xml]: I was going off of this document, which said to change XML attributes called "string": https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0 I can change it back, though.

Comment (lasley) [reply project_task_code/views/project_view.xml]: It's referring to string selectors, which means an [Xpath](http://www.odoo.yenthevg.com/xpath-expressions-in-odoo-8/) targetting a string attribute such as this one. The string attributes themselves aren't bad, just the use of the string as an Xpath injection point when inheriting.

Along those lines it is good to add name attributes to important objects like notebook pages that only have strings in order to allow for easier targeting. Not applicable here though 😄

Comment (kellylougheed) [reply project_task_code/views/project_view.xml]: OK, got it. I'll change it back!

**Review by lasley — APPROVED** [review]
Thanks @kellylougheed

**Review by pedrobaeza — APPROVED** [review]
Squashing before merging

**Review by dreispt — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@dreispt, I asked for a squash before merging. Now, there are 2 useless commits "Requested README changes" and "Requested XML change" on the repo...

**Pull request merged**
